source/MaterialXRenderGlsl/TextureBaker.cpp

```diff
@@ -103,73 +93,6 @@ TextureBaker::TextureBaker(unsigned int width, unsigned int height, Image::BaseT
     _frameCaptureImage->createResourceBuffer();
 }
 
-size_t TextureBaker::findVarInTemplate(const string& filename, const string& var, size_t start)
```

Comment:
For the clarity of reviewing this change and tracking it over time, I would recommend defining these functions in their original order within the source file, so that GitHub doesn't think they've all been deleted and then re-added in a different location.  It's fine if they're declared in a different, more canonical order in the header, since that establishes the public API for the class and its methods.